mobile-app/app/components/WalletTextInput.tsx

```diff
@@ -20,12 +29,12 @@ interface IWalletTextInputProps {
   displayFocusStyle?: boolean
   containerStyle?: string
   onBlur?: () => void
+  hasBottomSheet?: boolean
```

Comment:
Tried to look into the library to detect if there's a "wrapping" bottom sheet, but it isn't exposed anywhere